### PR TITLE
add recaptcha to queue route

### PR DESCRIPTION
**Pull request by guavajellyaaron**

### CODE REVIEW

File: netlify/functions/session.ts [hunk 1]
```diff
@@ -64,33 +57,21 @@ const handler: Handler = async (
...
+    return responseWithMessage(400, 'Invalid handle format.', true);
   }
 
-  await initFirebase();
-
   // Anti-bot.
   const reCaptchaValidated = await passesRecaptcha(headerRecaptcha);
-  if (!reCaptchaValidated) {
-    return unauthorizedResponse;
+  if (reCaptchaValidated) {
```
Comment (cjkoepke) [inline netlify/functions/session.ts:69]: @guavajellyaaron I think this needs to be `if( !reCaptchaValidated)`, right?

File: netlify/helpers/response.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        error: true,
+        message: "Unauthorized.",
+    }),
+};
+
+export const botResponse: HandlerResponse = {
+    statusCode: 422,
+    body: JSON.stringify({
+        message:
+            "Hmm, we think you might be a bot but we hope we're wrong. Please try again.",
```
Comment (cjkoepke) [inline netlify/helpers/response.ts:15]: Let's change this to `our system thinks you might...`

**Review by cjkoepke — APPROVED** [review]

File: netlify/functions/session.ts (hunk 1 above)
Comment (guavajellyaaron) [reply netlify/functions/session.ts:69]: Great catch!

**Review by cjkoepke — APPROVED** [review]
LGTM 🚀

**Pull request merged**